### ISSUE TITLE
Running the baseline: ppo.py

### ISSUE THREAD

**Issue by SeeYaaa** [proroklab/popgym#33]
Running the baseline ppo.py give me this warning:
WARNING algorithm_config.py:596 -- Cannot create PPOConfig from given `config_dict`! Property __stdout_file__ not supported.
should i ignore this?

**Comment by smorad** [comment]
Hmm that's strange. Are you using the correct version of RLlib (2.3.0 in `setup.cfg`)?

**Comment by SeeYaaa** [comment]
First of all Thank you very much sir for this repo! 
Quick answer: Yes! I can save and load the policy. The policy has a state in inference. So everything should work fine. 
And i don't get this warning with new version of ray, but i get some deprecations warnings, which are fine too :)

i followed your  instruction from your website https://popgym.readthedocs.io/en/latest/installation.html:
`conda create -n popgym python=3.9 -y
conda activate popgym

pip install popgym
pip install popgym[baselines]

python quick_start_baseline.py

--> ModuleNotFoundError: No module named 'tensorflow_probability'

pip install tensorflow-probability

python quick_start_baseline.py
 WARNING algorithm_config.py:597 -- Cannot create PPOConfig from given `config_dict`! Property __stdout_file__ not supported.

pip freeze | grep -E 'ray|torch|gymnasium|gym|popgym|python|tensorflow-probability'

gymnasium-notices==0.0.1
popgym==1.0.5
python-dateutil==2.9.0.post0
ray==2.3.0
tensorflow-probability==0.19.0
torch==1.13.1`

**Comment by smorad** [comment]
Sorry for the delay. I followed the same steps as you, except I ran the `ppo.py` script which is the entrypoint to the experiments.

```bash
conda create -n popgym python=3.9 -y
conda activate popgym
pip install popgym[baselines]
touch tensorflow_probability # RLlib has an import but doesn't actually use/require it
POPGYM_WORKERS=1 POPGYM_GPU=0 POPGYM_MODELS=GRU POPGYM_EXPERIMENT=PositionOnlyCartPoleEasy python ppo.py
```

I do end up getting the same warning

```python
(PPO pid=41768) 2024-03-15 17:05:30,947	WARNING algorithm_config.py:596 -- Cannot create PPOConfig from given `config_dict`! Property __stdout_file__ not supported.
```

However it appears to run fine on my machine. I took a look at the config it logged. Notice that the GRU model I chose is present in the config/run!

<img width="663" alt="Screenshot 2024-03-15 at 17 09 20" src="https://github.com/proroklab/popgym/assets/1743889/b6cf8bf0-ef56-437c-96e5-a282b655331c">

To summarize, I think this is just a deprecation warning from RLlib. You should be fine, but do let me know if you notice any training issues. I don't think it was present when I trained the models initially, but RLlib has been known to backport some updates to older versions. Perhaps that happened here?

**Comment by smorad** [comment]
Here is where RLlib throws this error: 

https://github.com/ray-project/ray/blob/3aa6ede43743a098b5e0eb37ec11505f46100313/rllib/algorithms/algorithm_config.py#L593-L600

It looks like the config dict will be used as expected, except for possibly the `__stdout_file__` (whatever that is).

**Comment by smorad** [comment]
Closing this as everything appears to work fine.
